### PR TITLE
[#5447][#5720][#5727] create topic, disable create table for hudi, enable schema properties and comment for partial paimon

### PR DESCRIPTION
**Pull request by LauraXia123**
### What changes were proposed in this pull request?

1. Support creating topic/editing topic/deleting topic/viewing topic
image

2. Fix issue with creating a table for hudi
image

3. [Improvement] Paimon catalog, schema properties can be set when backend is jdbc and hive
image
image

### Why are the changes needed?
N/A

Fix: #5447, #5720, #5727

### Does this PR introduce _any_ user-facing change?
N/A

### How was this patch tested?
manually

### CODE REVIEW

**Comment by LauraXia123** [comment]
@orenccl Hi, Could you help to review the PR when you have a moment?

**Comment by orenccl** [comment]
Hi @LauraXia123,

Sure, I’ll be able to review this PR either tonight or tomorrow night (UTC+8). Does that work for you?

File: web/web/src/app/metalakes/metalake/rightContent/CreateTopicDialog.js [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+                          <FormHelperText className={'twc-text-error-main'}>Key already exists</FormHelperText>
+                        )}
+                        {item.key && item.invalid && (
+                          <FormHelperText className={'twc-text-error-main'}>
+                            Invalid key, matches strings starting with a letter/underscore, followed by alphanumeric
+                            characters, underscores, hyphens, or dots.
+                          </FormHelperText>
+                        )}
+                        {!item.key.trim() && (
+                          <FormHelperText className={'twc-text-error-main'}>Key is required field</FormHelperText>
```
Comment (tengqm) [inline web/web/src/app/metalakes/metalake/rightContent/CreateTopicDialog.js:397]:
```suggestion
                          <FormHelperText className={'twc-text-error-main'}>Key is required</FormHelperText>
```

File: web/web/src/app/metalakes/metalake/rightContent/CreateTopicDialog.js [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+                          }}
+                        >
+                          {item.description}
+                        </FormHelperText>
+                        {item.hasDuplicateKey && (
+                          <FormHelperText className={'twc-text-error-main'}>Key already exists</FormHelperText>
+                        )}
+                        {item.key && item.invalid && (
+                          <FormHelperText className={'twc-text-error-main'}>
+                            Invalid key, matches strings starting with a letter/underscore, followed by alphanumeric
```
Comment (tengqm) [inline web/web/src/app/metalakes/metalake/rightContent/CreateTopicDialog.js:392]:
```suggestion
                            Valid key must starts with a letter/underscore, followed by alphanumeric
```

**Review by tengqm — COMMENTED** [review]
overall lgtm

File: web/web/src/app/metalakes/metalake/rightContent/CreateTopicDialog.js [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+  Dialog,
+  TextField,
+  Typography,
+  DialogContent,
+  DialogActions,
+  IconButton,
+  Fade,
+  Select,
+  MenuItem,
+  InputLabel,
```
Comment (orenccl) [inline web/web/src/app/metalakes/metalake/rightContent/CreateTopicDialog.js:37]: Select, MenuItem, and InputLabel seem unnecessary to import.

File: web/web/src/app/metalakes/metalake/rightContent/CreateTopicDialog.js [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+  const { open, setOpen, type = 'create', data = {} } = props
+  const searchParams = useSearchParams()
+  const metalake = searchParams.get('metalake')
+  const catalog = searchParams.get('catalog')
+  const schemaName = searchParams.get('schema')
+  const catalogType = searchParams.get('type')
+  const [innerProps, setInnerProps] = useState([])
+  const dispatch = useAppDispatch()
+  const store = useAppSelector(state => state.metalakes)
+  const activatedCatalogDetail = store.activatedDetails
```
Comment (orenccl) [inline web/web/src/app/metalakes/metalake/rightContent/CreateTopicDialog.js:91]: activatedCatalogDetail seems like an unnecessary variable.

**Review by orenccl — COMMENTED** [review]
LGTM. I've tested the topic dialog, and it works properly.

Thank you for fixing the issue caused by my earlier PR. However, we still need to add logic to handle user clicks on the edit table button when the catalog is Hudi.

Maybe we could display only the `Table` title instead of `Edit Table`, and prevent the user from changing values or clicking the update button, since it only supports read operations.

image

**Review by orenccl — APPROVED** [review]
Approved. Thank you for giving me the opportunity to review.

**Comment by LauraXia123** [comment]
@tengqm @orenccl Thanks for your time, I resolved all the issues based on the comment.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**